src/assets/body/tail/tail.asset.ts

```diff
@@ -0,0 +1,129 @@
+DefineBodypart({
+	name: 'Tail',
+	bodypart: 'tails',
+	allowRandomizerUsage: false,
+	graphics: 'graphics.json',
+	colorization: {
+		tail: {
+			name: 'Tail',
+			default: '#666666',
+		},
+		stripes: {
+			name: 'Stripes (not for all types)',
+			default: '#020202',
+		},
+		tip: {
+			name: 'Tip (not for all types)',
+			default: '#FFFFFF',
+		},
+	},
+	preview: null,
+	attributes: {
+		provides: [
+			'Tail',
+		],
+	},
+	modules: {
+		tailType: {
+			type: 'typed',
+			name: 'Tail Type',
```

Comment:
[suggestion, optional] Here again I wonder whether all these tails should be a single asset. Does it make sense to switch between the options while wearing it? Do they have a lot in common for this to make sense? I would say the thin, normal and thick options do... but what about the rest?
See my note on the plug variant for more details.